Utils/Dataflow/060_upload2virtuoso/uploadTTL.sh

```diff
@@ -163,7 +163,7 @@ upload_stream () {
   esac
 
   while true; do
-    while read -r -d "$delimiter" line; do
+    while read -r -d "$delimiter" line || [[ -n "$line" ]]; do
```

Comment:
Concerning your question: I think, if we want to work in "strict" mode (no EOM -- no message, no EOB -- no batch), then this change is not required.

Yet thank you for noticing: I have missed this issue when wrote the code.

By the way: if this commit was to be kept, there must be issue description in the commit log, right? ;)